uber/templates/common.js

```diff
@@ -1,18 +1,18 @@
-var csrf_token = "{% if CSRF_TOKEN %}{{ CSRF_TOKEN }}{% endif %}";
```

Comment:
I had to make a couple of small fixes in this file, so I converted the quotes while I was at it.